app/views/mentors/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Mentors</h1>
+
+<!-- Mentor Table -->
+<table class='highlight responsive-table'>
+  <thead>
+    <tr>
+      <th>Image</th>
+      <th>Name</th>
+      <th>Company</th>
+      <th>Specialty</th>
+      <th>Email</th>
+      <th>Phone</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% 15.times do %>   <!-- this will be changed once database created -->
+      <tr>
+        <td><%= image_tag 'user.jpg', class: 'demo-avatar' %></td>
+        <td><%= Faker::Name.name %></td>
+        <td><%= Faker::Company.name %></td>
+        <td><%= Faker::Job.field %></td>
+        <td><%= Faker::Internet.email %></td>
+        <td><%= Faker::PhoneNumber.cell_phone %></td>
+      </tr>
+    <% end %> 
+  </tbody>
+</table>
```

Comment:
very nice code. I love how you use the times() method here instead of just making a bunch of fake rows.. I never thought about doing like this. then I got thinking... maybe, lets take this one step further. we can code this so we do not need to change the view at all once the data base is in place. checkout it out.. we can fake the data. see comments on controller.
your view code might look like this:
```rails
  <tbody>
      <% @mentors.each do |mentor| %>
      <tr>
        <td><%= image_tag mentor["avatar"], class: 'demo-avatar' %></td>
        <td><%= mentor["name"] %></td>
        <td><%= mentor["company"] %></td>
        <td><%= mentor["specialty"] %></td>
        <td><%= mentor["email"] %></td>
        <td><%= mentor["phone"] %></td>
      </tr>
    <% end %> 
  </tbody>
```
now. this code will never have to change. it is done!
- What happens when @mentors is nil? and it will be at least once